Macedonian/macedonian.md

```diff
@@ -0,0 +1 @@
+Денес се лизнав на влажен под од плочки, но едно момче во инвалидска количка ме фати пред да треснам со главата на земја. Ми рече: 'Верувај или не но токму така го повредив грбот пред 3 години'. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.